fhir-server/liberty-config/configDropins/defaults/bulkdata.xml

```diff
@@ -41,15 +44,21 @@
             portNumber="50001"
             securityMechanism="15"
             serverName="${env.BATCH_DB_HOSTNAME}"
-            sslConnection="true" /> -->
-        <!-- <properties.postgresql
+            sslConnection="true" />
+    </dataSource>
+     -->
+   
+   <!-- Use postgresql for job repository  
+    <dataSource id="fhirbatchDS" jndiName="jdbc/fhirbatchDB">
+        <jdbcDriver libraryRef="fhirSharedLib"/>
+        <properties.postgresql
             databaseName="jobdb"
             portNumber="5432"
             serverName="localhost"
             user="javabatch"
-            password="change-password" /> -->
-        <properties.derby.embedded createDatabase="create" databaseName="derby/fhirDB"/>
+            password="change-password" />
     </dataSource>
+    -->
 
     <batchPersistence jobStoreRef="BatchDatabaseStore"/>
     <databaseStore dataSourceRef="fhirbatchDS" id="BatchDatabaseStore" schema="JBATCH" tablePrefix=""/>
```

Comment:
so it doesn't complain that `fhirbatchDS` doesn't exist by default?

Reply:
yes, it doesn't complaint on these 2 lines, so I kept them here, and so only need to define the datasource in the overrides

Reply:
and yes, I verified that derby is used in this case. the job id is always 1 after each restart of the server.

Reply:
OK, I havn't quite got around to test bulk export locally, but I'll take your word for it :-)

Reply:
thanks! :)